extensions/wikibase/module/styles/schema-alignment.css

```diff
@@ -49,12 +49,12 @@
 }
 
 .schema-alignment-dialog-entity-edits-area {
-  max-width: 70em;
+  max-width: 100%;
 }
 
 .schema-alignment-dialog-statements-area {
   padding: 10px;
-  max-width: 900px;
+  max-width: 100%;
```

Comment:
Why make this change?
The intention with capping the width of this element is to match the behaviour of Wikidata (where statements' width is also capped, see [this example on a wide screen](https://www.wikidata.org/wiki/Q69225951)).
I think it makes sense to cap the width because otherwise the buttons end up being very far from the content they relate to, so it makes it harder to follow which button corresponds to which statement/qualifier/reference.

Reply:
I understand your concern, but the image provided by Lozana in the issue #5449 (https://xd.adobe.com/view/3ae13acb-7afb-4ea5-8252-351a1622abba-6fb2/) had the buttons in the very corner of the screen, due to which I got the impression that the buttons had to be pushed to the end of the screen.
Please let me know if it needs to be reverted back to its original width.

Reply:
Intuitively the position of those buttons should be independent from the maximum width of the statements. You might need to change the DOM structure so that they are outside of this statements area.

Reply:
Okay got it

Reply:
On my side, the schema still expands to the full width of the screen:
![image](https://user-images.githubusercontent.com/309908/229113467-9195ee62-8428-4a67-8aec-7aa2e64602b6.png)

Reply:
Should I change the width back to its original length? I thought the schema should expand to the entire width of the screen.

Reply:
As I wrote above, I think it makes sense to cap the width because otherwise the buttons end up being very far from the content they relate to, so it makes it harder to follow which button corresponds to which statement/qualifier/reference. So yes, I would restore this back to its original length.